src/zac/demo/mor/tests/factories.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from factory.django import DjangoModelFactory as Dmf
+from zac.accounts.models import User
+from ...models import SiteConfiguration
+
+
+class UserFactory(Dmf):
+
+    class Meta:
```

Comment:
can you move the `class Meta` to below the field declarations (follow the pattern of Django models)?